bukkit/src/main/java/co/aikar/commands/ACFBukkitUtil.java

```diff
@@ -319,8 +319,8 @@ private static void findMatches(String search, CommandSender requester, List<Pla
     }
 
 
-    public static boolean isValidName(String name) {
-        return name != null && !name.isEmpty() && ACFPatterns.VALID_NAME_PATTERN.matcher(name).matches();
+    public static boolean isValidName(String name, CommandManager manager) {
```

Comment:
This is public therefore public api. There should be a backwards compatible method for those who use that method in their plugins.

Reply:
Done!